packit_service/models.py

```diff
@@ -128,6 +131,19 @@ def sa_session_transaction() -> SQLASession:
         raise
 
 
+def db_query(session: SQLASession, *entities: Any) -> Query:
+    """
+    Wrapper for SQLASession.query() that tries to access the connection first
+    and deal with potential PendingRollbackError.
+    """
+    try:
```

Comment:
could you please reference some docs/resources that you used?

Reply:
I've added a link to https://stackoverflow.com/a/69687698. Even though in our case the reason is different (I think it's because a rollback, even though issued, can't happen when postgres is unavailable), the solution could work for us.

Reply:
ok, thanks!